lib/entry/LiveEntry.js

```diff
@@ -144,8 +158,15 @@ LiveEntry.prototype.start = function() {
             return this.playlistGenerator.finalizeStart();
         })
         .then(()=> {
+            return getEntryServerNodeId.call(this)
```

Comment:
don't we already know the entryServerNodeId at this point?

Reply:
no

Reply:
Anyway moved code back to StateManager, though that is useless for recordingManager call to start recording

Reply:
Added a separate call to get EntryServerNodeId from RecordingEntrySession